file_organizer_v2/src/file_organizer/history/export.py

```
@@ -0,0 +1,294 @@
+
+        # Build export data
+        export_data = {
+            'export_date': datetime.utcnow().isoformat() + 'Z',
+            'operation_count': len(operations),
+            'operations': operations
+        }
+
+        # Include transactions if requested
+        if include_transactions:
+            # Get unique transaction IDs
+            transaction_ids = set(op.get('transaction_id') for op in operations if op.get('transaction_id'))
```

Ruff (0.14.13)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)